cmd/vm_scale.go

```diff
@@ -40,7 +39,7 @@ var vmScaleCmd = &cobra.Command{
 			}
 
 			if vm.State != (string)(egoscale.VirtualMachineStopped) {
-				fmt.Fprintf(os.Stderr, "this operation is not permitted if your VM is not stopped\n")
+				fmt.Fprintf(os.Stderr, "this operation is not permitted if your Compute instance is not stopped\n")
```

Comment:
[nitpicking] Use positive/guidance form :)
```suggestion
				fmt.Fprintf(os.Stderr, "this operation is not permitted while your Compute instance is not running; stop it before issuing that command again\n")
```